codes/quantum/qubits/stabilizer_over_gf4.yml

```diff
@@ -0,0 +1,27 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: stabilizer_over_gf4
+physical: qubits
+logical: qubits
+
+name: 'Stabilizer code over \(GF(4)\)'
+introduced: '\cite{arXiv:quant-ph/9608006}'
+
+description: 'An \([[n,k,d]]\) stabilizer code whose encoding is based on self-orthogonal \([n, n-k, d^*]\) code of \(GF(4)^n\) with respect to trace inner product where \(d \ge d^*\).'
```

Comment:
...over \(GF...

Reply:
Mention that this is based on the correspondence between elements of GF(4) and qubit Pauli operators.

Reply:
Updated